pulp_ostree/tests/functional/conftest.py

```diff
@@ -97,3 +118,27 @@ def _ostree_distribution_factory(**body):
         return gen_object_with_cleanup(ostree_distributions_api_client, data)
 
     return _ostree_distribution_factory
+
+
+# Utils
+
+
+@pytest.fixture(scope="class")
+def synced_repo_version(
+    ostree_repositories_api_client,
+    ostree_repositories_versions_api_client,
+    ostree_repository_factory,
+    ostree_remote_factory,
+    monitor_task,
+):
+    """
+    Fixture that syncs a Remote ostree Repository,
+    waits until the sync task completes and returns the
+    created repository version object.
+    """
+    repo = ostree_repository_factory()
+    remote = ostree_remote_factory(url=OSTREE_FIXTURE_URL, policy="immediate")
+    result = ostree_repositories_api_client.sync(repo.pulp_href, {"remote": remote.pulp_href})
+    repo_version_href = monitor_task(result.task).created_resources[0]
+    repo = ostree_repositories_api_client.read(repo.pulp_href)
+    return ostree_repositories_versions_api_client.read(repo_version_href), remote, repo
```

Comment:
It might be worth returning the updated repository, containing a new latest repository version after syncing.
```
repo = ostree_repositories_api_client.read(repo.pulp_href)
```

Reply:
Oh, my bad. This one is correct. It went unnoticed.